jooby/src/main/java/io/jooby/internal/RouterImpl.java

```diff
@@ -909,7 +909,7 @@ private void copy(Route src, Route it) {
     it.setTags(src.getTags());
     it.setDescription(src.getDescription());
     it.setDecoders(src.getDecoders());
-    it.setMvcMethod(it.getMvcMethod());
+    it.setMvcMethod(src.getMvcMethod());
```

Comment:
@jknack this line still look like a type, should I close the PR or you are going to merge?

Reply:
good catch! keep it, will merge later